Update link at the bottom of the Work Order Details page

mateoclarke opened the pull request:
Closes #213 

![Sep-06-2019 16-03-31](https://user-images.githubusercontent.com/5697474/64460419-f472b580-d0bf-11e9-919e-0910aea12c83.gif)

mateoclarke committed:
Update link at the bottom of the Work Order Details page

dianamartin commented:
Your video looks so good! I think it's working properly. If we could make it the same size and bold text that we have for the "<-- Back to View Work Order Details" on the location details page so it's consistent. 

https://atd.knack.com/24-july-2019--test-signs-and-markings-operations#my-work-orders/view-work-orders-details-sign/5d3ef9fa2bd5fd0011339b7e/view-work-order-signs-location-details/5d6830a8d804050010676ec8/

![image](https://user-images.githubusercontent.com/21112975/64462031-b1ffa780-d0c4-11e9-8d0f-bdbf811d02dd.png)

mateoclarke committed:
update back to top styling. bold, bigger and icon

mateoclarke commented:
@dianamartin I've updated the "Back to top" link to match the "Back to View Work Order Details" link. Went ahead and added an up arrow too.

Going to merge now with this update.

![Screen Shot 2019-09-06 at 4 53 51 PM](https://user-images.githubusercontent.com/5697474/64462767-2b989500-d0c7-11e9-9f51-a0d9acb754e5.png)
